be/src/vec/exec/format/parquet/level_decoder.h

```diff
@@ -27,6 +27,8 @@
 
 namespace doris::vectorized {
 
+static const size_t V1_LEVEL_SIZE = 4;
```

Comment:
Use constexpr. It's better to rename to `V1_SIZE_LENGTH`.
The the const variable should be a static member of LevelDecoder, or move it to cpp file, otherwise it will be a global variable in `doris::vectorized` namespace.